tcms/rpc/api/testcase.py

```diff
@@ -255,6 +255,8 @@ def create(values, **kwargs):
         result = model_to_dict(test_case, exclude=["component", "plan", "tag"])
         # b/c date is added in the DB layer and model_to_dict() doesn't return it
         result["create_date"] = test_case.create_date
+        result["setup_duration"] = str(result["setup_duration"])
+        result["testing_duration"] = str(result["testing_duration"])
```

Comment:
Why do you need to explicitly convert to string here ?

Reply:
I did that because xmlrpc doesn't know how to marshal datetime.timedelta objects, and throws an error if I don't convert the durations to strings.

Reply:
Ah right. I had forgotten about this fact.